Cython/Compiler/Tests/TestTreeFragment.py

```diff
@@ -59,6 +60,26 @@ def test_temps(self):
         self.assertTrue(isinstance(s[1].rhs, TempRefNode))
         self.assertTrue(s[0].expr.handle is s[1].rhs.handle)
 
+    def test_parse_fault_tolerant_complete(self):
+        code = u'''
+def method():
+    pass
+
+def method2():
+    error_here =
+
+def method3():
+    error_here =
+
+def method4():
+    without_error_here = 10
```

Comment:
I think that the last statement should also be an error (say `error_here =`) to see how it behaves in that corner case.